overlord/overlord.go

```diff
@@ -102,6 +103,7 @@ func New() (*Overlord, error) {
 	o.hookMgr = hookMgr
 	o.stateEng.AddManager(o.hookMgr)
 
+	hooks.SetupHooks(o.hookMgr)
```

Comment:
Unfortunately we can't do it from snapstate's Manager (or I'm missing something?). Doing it from the Manager requires hookstate import and this creates import cycle (since hookstate imports snapstate).